core/src/main/java/io/questdb/griffin/engine/table/AbstractDescendingRecordListCursor.java

```diff
@@ -71,6 +73,17 @@ void of(DataFrameCursor dataFrameCursor, SqlExecutionContext executionContext) t
         this.recordB.of(dataFrameCursor.getTableReader());
         rows.clear();
         buildTreeMap(executionContext);
+        this.isOpen = true;
```

Comment:
`LatestByValuesFilteredRecordCursor` and `LatestByValuesRecordCursor` do not use reallocation parttern